.env

```diff
@@ -86,4 +86,33 @@ _APP_GRAPHQL_MAX_COMPLEXITY=250
 _APP_GRAPHQL_MAX_DEPTH=3
 DOCKERHUB_PULL_USERNAME=
 DOCKERHUB_PULL_PASSWORD=
-DOCKERHUB_PULL_EMAIL=
\ No newline at end of file
+DOCKERHUB_PULL_EMAIL=
+VCS_GITHUB_NAME=appwrite-vcs
+VCS_GITHUB_PRIVATE_KEY="-----BEGIN RSA PRIVATE KEY-----
+MIIEpQIBAAKCAQEA6U8z6jGzlEOHmrCmyzlb5e4Tl4A7PFs5h2RkTrTWNBjmOsOT
```

Comment:
I think you should revoke your private access key as soon as possible.

Reply:
It's an inactive key.